scripts/boto3/ultihash_info/get_effective_size.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Jun 10 11:22:39 2024
+
+@author: massi
+"""
+
+import argparse
+import boto3
+import botocore
+import json
+
+AWS_KEY_ID="key-id"
+AWS_KEY_SECRET="secret"
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        prog='UH Stats',
+        description='Statistics from UH cluster')
+
+    parser.add_argument('-u', '--url', help='override default S3 endpoint',
+        default='http://localhost:8080', dest='url')
+
+    return parser.parse_args()
+
+def get_effective_size(config):
+    s3_cnf = botocore.config.Config(
+        read_timeout=10,
+        retries = {
+            'max_attempts': 3,
+            'mode': 'standard'
+        })
+
+    s3 = boto3.client('s3', endpoint_url=config.url, config=s3_cnf,
+            aws_access_key_id=AWS_KEY_ID, aws_secret_access_key=AWS_KEY_SECRET)
+    response = s3.get_object(Bucket="ultihash", Key="v1/metrics/cluster")
```

Comment:
I am seriously surprised this works as the response is definitely not S3 conformant... But if boto accepts it, I am also happy ;-)